content/slack.md

```diff
@@ -5,5 +5,5 @@ aliases:
     - "/slack.html"
 ---
 
-Redirecting to slack. If it doesn't happen automatically, click <a href='https://join.slack.com/t/valkey-oss-developer/shared_invite/zt-2nxs51chx-EB9hu9Qdch3GMfRcztTSkQ'>here</a>.
+Redirecting to slack. If it doesn't happen automatically, click this <a href='https://join.slack.com/t/valkey-oss-developer/shared_invite/zt-2nxs51chx-EB9hu9Qdch3GMfRcztTSkQ'>invitation</a>.
```

Comment:
Note that this link doesn't work...
- https://github.com/valkey-io/valkey/issues/2155